video2numpy/read_vids_cv2.py

```diff
@@ -63,8 +82,39 @@ def get_frames(vid):
             "dst_name": dst_name,
             "pad_by": pad_by,
         }
-        queue.put(np_frames, info)
+        queue_video.put(np_frames, info)
+
+
+    def get_audio(vid):
+        if not vid.endswith(".wav"):
+            load_vid, dst_name = handle_youtube(vid, "audio")
```

Comment:
I don't think we need two separate handle_youtube calls. Instead of making get_frames, get_audio try to unify them into get_video that takes the ```modalities``` parameter and also pass that parameter into the handle_youtube function and return everything needed. Less code is better